### ISSUE TITLE
How I can change length text to  generate QR Code

### ISSUE THREAD

**Issue by giggstae** [witwall/barcode-vba-macro-only#1]
I load sample Excel but my data cannot generate full text

sample:

Text : ODH080014$OL1$PRM20H0013$Q34$LSTOREPC2$SRM1711200001
(in 1 line)

QR generate data only : ODH080014$OL1$PRM20H0013$Q34$LSTOREPC2$SRM

Please help suggest where the code I can change.

**Comment by JonasHeidelberg** [comment]
I think your problem might be related to the “stuttering“ I observed and fixed in my fork https://github.com/jonasheidelberg/barcode-vba-macro-only

Generally these problems are likely to have to do with switching the encoding within the QR code (in your case the switch from captial letters to numbers-only does not happen).

Could you check if my fork also shows your problem?

Best
Jonas

**Comment by giggstae** [comment]
I download sample excel file follow this picture.

If I incorrect, please share a correct link.

![question](https://user-images.githubusercontent.com/30170788/33157452-c7d65186-d034-11e7-9eff-7565716b6cf4.png)

and this is my file
![image](https://user-images.githubusercontent.com/30170788/33157494-0c96dde0-d035-11e7-8535-ed8296da7291.png)

**Comment by JonasHeidelberg** [comment]
You are right, the error you had was different from the one I fixed in February. Sorry for that! The code generating the eb() array is just deeply broken, I'd love to be able to talk to the original author to at least understand it better, this would greatly simplify debugging.

**I will upload a quickfix for your problem in a few minutes.** I hope this covers all use cases that you have, but I cannot be sure the code is now bug-free, because I don't have the time to set up a serious testing suite and/or rewrite the buggy code completely.

**Comment by giggstae** [comment]
It's work.

Thank you very much.
